blank-canvas-blocks/inc/block-patterns.php

```diff
@@ -0,0 +1,503 @@
+									<!-- wp:paragraph -->
+									<p>' . esc_html__( 'Join us for a celebratory party featuring creative cocktails and mocktails, seasonal chef-crafted snacks and appetizers, and a range of desserts and pastries from local bakeries.', 'blank-canvas-blocks' ) . '</p>
+									<!-- /wp:paragraph -->
+									<!-- wp:buttons -->
+									<div class="wp-block-buttons"><!-- wp:button {"style":{"color":{"background":"#c61aa7"}},"textColor":"background","className":"is-style-fill"} -->
+									<div class="wp-block-button is-style-fill"><a class="wp-block-button__link has-background-color has-text-color has-background" style="background-color:#c61aa7">' . esc_html__( 'RSVP', 'blank-canvas-blocks' ) . '</a></div>
+									<!-- /wp:button --></div>
+									<!-- /wp:buttons -->
+									<!-- wp:spacer {"height":30} -->
+									<div style="height:30px" aria-hidden="true" class="wp-block-spacer"></div>
+									<!-- /wp:spacer --></div></div>
+									<!-- /wp:media-text -->',
+				)
+			);
+
+			register_block_pattern(
+				'blank-canvas-blocks/split-screen',
+				array(
+					'title'      => __( 'Split Screen', 'blank-canvas-blocks' ),
+					'categories' => array( 'blank-canvas-blocks' ),
+					'content'    => '<!-- wp:media-text {"align":"full","mediaId":598,"mediaLink":"' . esc_url( get_stylesheet_directory_uri() . '/assets/images/pattern-camera.jpg' ) . '","mediaType":"image","imageFill":true} -->
+									<div class="wp-block-media-text alignfull is-stacked-on-mobile is-image-fill"><figure class="wp-block-media-text__media" style="background-image:url(' . esc_url( get_stylesheet_directory_uri() . '/assets/images/pattern-camera.jpg' ) . ');background-position:50% 50%"><img src="' . esc_url( get_stylesheet_directory_uri() . '/assets/images/pattern-camera.jpg' ) . '" alt="' . esc_attr__( 'Photo of a vintage camera.', 'blank-canvas-blocks' ) . '" class="wp-image-598 size-full"/></figure><div class="wp-block-media-text__content"><!-- wp:cover {"overlayColor":"white","minHeight":95,"minHeightUnit":"vh"} -->
+									<div class="wp-block-cover has-white-background-color has-background-dim" style="min-height:95vh"><div class="wp-block-cover__inner-container"><!-- wp:image {"align":"center","id":624,"width":200,"height":200,"sizeSlug":"large","linkDestination":"none","className":"is-style-rounded"} -->
+									<div class="wp-block-image is-style-rounded"><figure class="aligncenter size-large is-resized"><img src="' . esc_url( get_stylesheet_directory_uri() . '/assets/images/pattern-woman.jpg' ) . '" alt="' . esc_attr__( 'Photo of a woman.', 'blank-canvas-blocks' ) . '" class="wp-image-624" width="200" height="200"/></figure></div>
+									<!-- /wp:image -->
+
+									<!-- wp:heading {"textAlign":"center","level":1,"textColor":"almost-black"} -->
+									<h1 class="has-text-align-center has-almost-black-color has-text-color"><strong>'. esc_html__( 'Jasmine Baker', 'blank-canvas-blocks' ) . '</strong></h1>
```

Comment:
I think we could use just the hex color but this is not a universal block pattern, is it? according to our slack conversation, child themes are going to unregister these.

Reply:
I don't understand how all block patterns aren't "universal".  If on theme A I use a block pattern, then change themes the blocks are still there but they they (potentially) look bad because classes that were expected are no longer there.

Reply:
yeah, that's the problem if we don't use hex codes or until we get semantic colors that are available to all themes...

Reply:
Yeah, I think for now we should use hex colors in all of the Blank Canvas patterns to combat this.